rts/motoko-rts/src/bigint.rs

```diff
@@ -246,6 +246,25 @@ unsafe extern "C" fn bigint_of_int64(j: i64) -> SkewedPtr {
     persist_bigint(i)
 }
 
+#[cfg(feature = "ic")]
+#[no_mangle]
+unsafe extern "C" fn bigint_of_float64(j: f64) -> SkewedPtr {
+    let mut i = tmp_bigint();
+    mp_set_double(&mut i, j);
```

Comment:
check result here

Reply:
Done in 06c038331. But I see several other places where we don't check yet, e.g. `mp_set_i64/u64`. @osa1

Reply:
I checked the documentation and the source, I think `mp_set_i64` and `mp_set_u64` cannot fail.